sdk/core/core-tracing/src/tracerProxy.ts

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
seems like we got a lot of duplicate license headers...

Reply:
thank you, Jeff! I will clean this up in a bit